src/main/java/user/createcorporate/CorporateGenerateRequestModel.java

```diff
@@ -1,24 +1,20 @@
 package user.createcorporate;
 
+/** A request model for the createcorporate use case that frames the input data into an object.
+ * Holds a String representation of the Id of the user who manages the corporate in userId, a
+ * String of the company's name in companyName, a String of a summary of the corporate intro
+ * in companyInfo.
+ */
```

Comment:
same as above, it could be even better if you directly edit the lines 10-16 that you deleted :)

Reply:
Same as above